### DISCUSSION TITLE
How to avoid Ant Design overriding existing CSS?

### DISCUSSION THREAD

**Discussion by tungvu2210** [ant-design/ant-design#34644] [Q&A] [answered]
I am doing a ReactJS project. At first, the project was only written in pure CSS, then I wanted to use a small Antd component for my project. I followed the instructions and imported like this into a specific ReactJS component:

```
import { Pagination } from 'antd';
import 'antd/dist/antd.css';
```

The issue is that my existing CSS is overridden by Ant Design. Can you please give me a solution?

**Comment by yoyo837** [comment, marked as answer]
Caused by this? https://github.com/ant-design/ant-design/issues/9363
